src/SkiddleBase.php

```diff
@@ -125,24 +136,116 @@ public function formatForLog($data = [], $unset = false)
                     unset($data[$k][$old]);
                 }
             }
-            foreach ($reformat_venues as $old => $new) {
+            foreach ($reformatVenues as $old => $new) {
                 $data[$k][$new] = $ticket['venue'][$old];
                 if ($unset) {
                     unset($data[$k]['venue'][$old]);
                 }
             }
             $data[$k]['Town'] = $ticket['venue']['town']; //:thinking_face:
-            foreach ($reformat_times as $old => $new) {
+            foreach ($reformatTmes as $old => $new) {
                 $data[$k][$new] = $ticket['openingtimes'][$old];
                 if ($unset) {
                     unset($data[$k]['openingtimes'][$old]);
                 }
+                $data[$k][$new] = $ticket['openingTimes'][$old];
+                if ($unset) {
+                    unset($data[$k]['openingTimes'][$old]);
+                }
             }
         }
 
         return $data;
     }
 
+    /**
+     * v3 API - helper function
+     * @param array  $responseData event data in the response
+     * @param array  $renameKeys keys to remap
+     * @return array event data with transformed keys
+     */
+    private function recursiveRenameKeys(array $array, array $renameKeys): array
```

Comment:
I'd rename the first property to something more meaningful. Making it match your docblocks makes sense. This removes the need for your annotations too.


```suggestion
    private function recursiveRenameKeys(array $responseData, array $renameKeys): array
```

Don't forget `$array` on line 171 too :)